src/main/kotlin/baseball/Application.kt

```diff
@@ -1,5 +1,96 @@
 package baseball
 
+import camp.nextstep.edu.missionutils.Randoms.pickNumberInRange
+
 fun main() {
-    TODO("프로그램 구현")
+    var oneOrTwo: Int = 1
+    var myAnswer: String
+    var comAnswer: String
+
+    println("숫자 야구 게임을 시작합니다.")
+
+
+    while(oneOrTwo==1){
+        comAnswer = comNumberCreator()
+        print("숫자를 입력해주세요 : ")
+        myAnswer = readLine()!!.toString()
```

Comment:
요구사항 보시면 Console.readLine()을 통해 입력을 받으라고 되어있습니다!

별개로 readLine()은 nullable이므로 !!보다는 

```suggestion
        myAnswer = readLine() ?: ""
```
이렇게 쓰시는게 런타임 오류를 방지하기 좋습니다.